src/main/java/com/gsheet/student/transformer/StudentToStudentDtoTransformer.java

```diff
@@ -0,0 +1,17 @@
+package com.gsheet.student.transformer;
+
+import com.gsheet.student.dto.StudentDto;
+import com.gsheet.student.entity.Student;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Transformer is not a service, you should use static methods on the transformer indeed